docusaurus.config.js

```diff
@@ -104,12 +104,12 @@ module.exports = {
         },
         {
           to: "build/",
-          label: "Build on Celo",
+          label: "Get Started",
           position: "left",
         },
         {
           to: "developer/",
-          label: "Developers",
+          label: "Build on Celo",
           position: "left",
```

Comment:
why do we not call this section tools? The top page in this section is called Tools.